bin/cli.js

```diff
@@ -223,3 +229,50 @@ function processWatch(environment) {
   compiler.watch({}, (err, stats) => {
   });
 }
+
+
+function processTest() {
+  runTest();
+}
+
+function processTestWatch() {
+  runTest({ watch: true });
+}
+
+function runTest({ watch } = {}) {
+  const createMochaWebpack = require('mocha-webpack');
+  const testHelper = require('../helpers/testHelper');
+  const coverageHelper = require('../helpers/coverageHelper');
+  const combinedReporter = require('../helpers/combinedTestReporter');
+
+  const mochaWebpack = createMochaWebpack();
+
+  const reportDirectory = path.resolve(process.cwd(), './Resources/Private/Javascript/coverage');
+
+  testHelper.prepareTestEnvironment();
+  if (!watch) {
+    coverageHelper.prepareCoverageReporter();
+  }
+
+  mochaWebpack.cwd(process.cwd());
+  mochaWebpack.webpackConfig(require('../webpack.test')('development', argv));
+  mochaWebpack.addEntry(path.resolve(process.cwd(), '../Customer.Base/Resources/Private/Javascript/') + '/**/*.test.js');
```

Comment:
Foldername is a placeholder here
```suggestion
  mochaWebpack.addEntry(path.resolve(basePackagePathAbsolute() ,'./Resources/Private/Javascript/') + '/**/*.test.js');
```